<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_interface</class>
 <widget class="QWidget" name="login_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_log_log">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_log_back">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_log_reg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>【没有账号？点击注册】</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>   未连接服务器</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_log_acct">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_log_pwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>95</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
